source/webgl-operate.ts

```diff
@@ -13,6 +13,13 @@ export import fetch = root_fetch;
 import * as root_raymath from './raymath';
 export import ray_math = root_raymath;
 
+export { FontFace } from './fontface';
+export { FontLoader } from './fontloader';
+export { GlyphVertex, GlyphVertices } from './glyphvertices';
+export { Label } from './label';
+export { LabelGeometry } from './LabelGeometry';
```

Comment:
I changed the file name using lower case, as you suggested.

Reply:
done